quantumdb-cli/src/main/java/io/quantumdb/cli/xml/XmlAddForeignKey.java

```diff
@@ -12,28 +12,32 @@
 @Data
 public class XmlAddForeignKey implements XmlOperation<AddForeignKey> {
 
-	static final String TAG = "addForeignKey";
+	static final String TAG = "createForeignKey";
```

Comment:
Since the XML ns is leading, the TAG is changed to createForeignKey. This will have to be changed when a new version is made.